packages/core/src/components/TabsSlider/Tabs.tsx

```diff
@@ -0,0 +1,51 @@
+import { WithStyle } from '@medly-components/utils';
+import React, { useCallback, useMemo, useState } from 'react';
+import Tab from './Tab';
+import TabList from './TabList';
+import TabPanel from './TabPanel';
+import * as Styled from './Tabs.styled';
+import { Props, StaticProps } from './types';
+
+export const Tabs: React.FC<Props> & StaticProps & WithStyle = React.memo(
```

Comment:
I guess we have to give it a different name so that we can export it in the `components/index.ts`